resources/views/layouts/header.blade.php

```diff
@@ -1,3 +1,11 @@
+@php 
+
+    use Illuminate\Support\Facades\Route;
+
+    $routeName = Route::currentRouteName();
```

Comment:
Oh, clever. Please share this insight with the other teams who might be having problems with this.